### PR TITLE
Feat: Bun 1.0 runtime

### PR DESCRIPTION
**Pull request by Meldiron**
This adds Bun runtime into Open Runtimes. Later it can be used in Appwrite.

Tasks:

- [x] Create runtime from Deno (take from Node what is needed)
- [x] Use Deno HTTP server
- [x] Implement tests

Related issue:

* https://github.com/appwrite/appwrite/issues/3538

### CODE REVIEW

File: runtimes/bun-1.0/helpers/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Fail build if any command fails
+set -e
+
+. /usr/local/server/helpers/before-build.sh
```
Comment (christyjacob4) [inline runtimes/bun-1.0/helpers/build.sh:5]: what is this `.` supposed to be doing?

**Review by christyjacob4 — APPROVED** [review]
please check the comment

File: runtimes/bun-1.0/helpers/build.sh (hunk 1 above)
Comment (stnguyen90) [reply runtimes/bun-1.0/helpers/build.sh:5]: it's the same as [`source`](https://linuxize.com/post/bash-source-command/) which is kind of like an import and execute the script in the current context.

Comment (Meldiron) [reply runtimes/bun-1.0/helpers/build.sh:5]: Yep, it's replacement for source. We use it in ~half of runtimes. Some (for some reason) simply don't have source. `.` is drop-in replacement, seems like. So far I didn't spot any issue with it

Comment (Meldiron) [reply runtimes/bun-1.0/helpers/build.sh:5]: For example in Java: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/java-18.0/helpers/build.sh

Comment (christyjacob4) [reply runtimes/bun-1.0/helpers/build.sh:5]: Alright got it 👍

**Pull request merged**